omod/src/main/webapp/fragments/field/concept.gsp

```diff
@@ -0,0 +1,58 @@
+<%
+    config.require("formFieldName")
+    config.require("conceptId")
+%>
+
+<script type="text/javascript">
+    jq(function() {
+        var xhr=null;//used to track active ajax requests
+        
+        jq('#${ config.id }-field').autocomplete({
+            source: function(request, response) {
+                var ajaxUrl = '${ ui.actionLink("registrationapp", "concept", "getConcepts")}';
```

Comment:
Yes, the list of answers is a field on concept, you have the concept in the page scope at the time of rendering the gsp, why not create a JS variable and store then rather going back to the server?